config/application.rb

```diff
@@ -73,5 +72,9 @@ class Application < Rails::Application
                                    key: 'api_session',
                                    secure: Settings.session_cookie.secure,
                                    http_only: true
+
+    # These files do not contain auto-loaded ruby classes,
+    #   they are loaded through app/workers/education_form/forms/base.rb
+    Rails.autoloaders.main.ignore(Rails.root.join('app', 'workers', 'education_form', 'templates', '1990-disclosure'))
```

Comment:
Seems like we'd either have none of these or lots of these. One seems suspicious.

Reply:
Interesting point.  These are the only ones that fail tests.  I'll look around for some more

Reply:
Is there another way to prevent the failures but allow autoloading? Maybe that's what the others are doing.

Reply:
Ah, part of the problem is those files were named with `_CH30.erb` (or other numbers) and there isn't a good way to inflect these to satisfy the loader.  But since they aren't dynamic ruby classes, I think ignoring them here was the best approach.

Reply:
Here's the ticket surrounding for this https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/15891

Reply:
Other places with the `File.read` + `ERB.new` pattern have files that match the autoloading conventions.  But perhaps they should be excepted here.  I think it's largely moot though.